src/include/func_specific/rng_seed_stream.hpp

```diff
@@ -5,7 +5,7 @@
 565122, 218912, 174581, 196530, 344741, 228066, 152552, 207187, 661837, 641858, 481710, 412472, 899531, 611904, 804070, 413828, 579413, 121217, 926830, 958460, 233749, 122945, 442811, 116950, 609635, 787973, 384577, 866458, 126662, 337845, 813330, 951269, 458065, 430145, 281077, 914761, 118870, 118758, 543997, 103119, 151890, 623045, 287276, 390815, 912083, 746546, 369251, 498572, 436475, 731575, 467335, 880680, 550970, 101179, 354007, 206907, 294659, 269960, 390099, 847256, 572100, 527174, 897499, 356447, 222243, 770882, 498668, 415504, 981412, 459041, 489495, 263604, 442372, 183142, 312301, 310171, 839561, 402933, 405053, 674727, 579473, 160523, 778354, 652371, 113083, 161188, 396032, 524845, 227135, 933566, 560288, 153120, 160999, 761551, 125900, 146991, 383414, 839055, 909592, 442128, 189475, 892829, 937923, 180717, 548304, 896337, 603797, 708338, 965674, 788503, 151747, 966881, 314313, 514023, 612578, 774008, 937090, 647423, 113452, 110740, 481173, 204790, 996663, 255529, 212060, 253106, 194450, 856201, 193931, 110041, 546492, 413970, 186401, 723026, 248096, 195236, 372531, 930979, 846335, 188331, 261509, 131754, 182146, 184919, 642664, 815748, 261083, 916165, 134566, 382777, 780349, 798391, 715996, 224934, 980428, 156734, 187955, 156677, 773302, 249917, 125113, 639087, 202505, 311563, 672347, 426212, 362505, 743621, 987095, 842631, 188314, 362879, 660800, 229360, 177005, 823421, 142148, 401458, 573307, 902485, 187445, 745262, 646020, 264540, 718913, 888389, 425034, 202593, 730165, 823966, 167154, 279338, 981778, 204504, 943552, 624664, 152453, 276996, 949596, 146852, 393163, 521221, 730615, 300406, 983918, 480434, 203775, 202217, 192421, 244059, 600289, 790584, 626339, 953715, 221773, 164676, 536715, 461364, 188559, 871765, 535738, 162355, 266215, 198835, 478630, 155259, 764716, 235733, 481317, 306885, 732218, 115590, 917449, 723326, 324296, 878162, 805300, 865740, 830788, 319344, 297076, 630370, 386384, 780253, 100584, 731605, 134795, 820104, 121115, 514356, 123256, 400428, 446013, 151280, 916043, 345287, 379787, 129565, 947156, 974294, 707583, 910553, 130917, 981660, 146049, 793212, 651934, 537031, 168055, 823490, 806687, 503217, 471696, 593786, 102956, 558731, 276764, 643359, 289622, 957489, 619223, 651677, 191795, 283771, 155046, 186852, 310593, 121445, 343704, 153587, 293532, 380375, 321084, 595166, 161268, 944827, 542115, 316626, 428468, 130162, 684777, 542142, 998308, 150446, 575299, 340353, 977809, 837318, 734692, 613815, 214647, 462703, 226586, 863995, 563267, 734383, 573703, 891686, 973411, 879183, 590344, 150313, 747737, 968821, 200808, 589146, 231348, 533529, 575375, 256785, 403840, 775818, 939712, 618134, 539406, 267952, 176201, 267510, 414783, 177672, 152609, 702173, 198224, 547158, 469752, 507664, 911999, 449574, 894131, 720221, 446580, 120735, 231717, 980608, 850706, 132276, 944840, 590995, 582994, 761474, 957166, 239989, 309587, 190973, 836071, 381133, 124200, 711759, 334858, 495950, 590018, 362332, 615438, 179266, 9505798
```

Comment:
avoid using so wide statements. Please split it into multiple lines for readability

Reply:
Done